tutorials/napps/development_environment_setup.rst

```diff
@@ -160,6 +160,10 @@ and run:
   $ source test42/bin/activate
   $ kytosd -f
 
+.. NOTE:: It may be necessary to install some NApp dependencies like
+    `filelock`, to install use `pip install filelock==3.0.12` inside the
+    virtualenv.
```

Comment:
I think it is better to turn this note in an instruction of the tutorial. What happens if the user did not install the dependencies and try using env? It will not work, right?

Reply:
I agree with Mauro. Also, the user can install directly from the napps' requirements/dev.txt, that has all the dependencies.

Reply:
Resolved on the commit https://github.com/kytos/documentation/pull/106/commits/d77c5ac95ee6311b71526fb3b701b2a05598d72f

Reply:
More information related to this step was added in https://github.com/kytos/documentation/pull/106/commits/8de12b8f1b8e7bd0705c1c4ef8d2536cd359345e